src/riak_kv_crdt.erl

```diff
@@ -507,13 +522,13 @@ eqc_test_() ->
      ?_test(?TIMED_QC(prop_binary_roundtrip()))}.
 
 prop_binary_roundtrip() ->
-    ?FORALL({_Type, Mod}, oneof(?MOD_MAP), 
+    ?FORALL({_Type, Mod}, oneof(?MOD_MAP),
```

Comment:
Yeah... for larger diffs w/ older files, I'll make sure to turn it off. I also believe trailing whitespace is "bad."